### ISSUE TITLE
Incorrect comment on the top of train_mobilenetdet_on_kitti.sh

### ISSUE THREAD

**Issue by KirillDyagilev** [Zehaos/MobileNet#23]
"
# This script performs the following operations:
# 1. Trains a MobileNet model on the Imagenet training set.
# 2. Evaluates the model on the Imagenet validation set.
"
https://github.com/Zehaos/MobileNet/blob/master/scripts/train_mobilenetdet_on_kitti.sh

Thank you for making these scripts available!

**Comment by Zehaos** [comment]
Thanks. : )
